Examples/UIExplorer/js/TextExample.ios.js

```diff
@@ -465,6 +465,33 @@ exports.examples = [
       </View>
     );
   },
+}, {
+  title: 'Font variants',
+  render: function() {
+    return (
+      <View>
+        <Text style={{fontVariant: ['small-caps']}}>
+          Small Caps{'\n'}
+        </Text>
+        <Text style={{fontFamily: 'Hoefler Text', fontVariant: ['oldstyle-nums']}}>
```

Comment:
These fonts won't generally be available on peoples phones or simulators. Can you pick another one?